cargo-near-build/src/near/build/mod.rs

```diff
@@ -55,6 +55,14 @@ pub fn run(args: Opts) -> eyre::Result<CompilationArtifact> {
         )
     })?;
 
+    // addition of this check wasn't a change in logic, as previously output path was
+    // assumed without `--out-dir` too, so docker-build was just failing if the arg was supplied:
```

Comment:
log output before when attempting to use `--out-dir`: 
```bash
✓ Running docker command step (finished)
• Copying artifact from temporary build site...
 artifact search location in temporary build site: /tmp/.tmplMHMxh/target/near
• failed

Here is the console command if you ever need to re-run it again:
cargo near build reproducible-wasm

Error:
   0: Temporary build site result wasm file not found: `"/tmp/.tmplMHMxh/target/near/simple_package.wasm"`.

Location:
   cargo-near-build/src/types/near/docker_build/cloned_repo.rs:134
```

Reply:
I don't understand why we must forbid `--out-dir`. Just let it fail to find the wasm file at the end of the execution and force the user to provide the path explicitly in the `[package.metadata.near.reproducible_build]` section.

Are you trying to minimize the number of configurable options in the `[package.metadata.near.reproducible_build]` section? If that is the case, I get it, and it is fine to move forward like this. However, it might be more flexible to have a way to allow users to specify where to find the wasm file.